tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDisplayAndInputProcessor.java

```diff
@@ -576,4 +576,17 @@ public void TestSplitBar() {
         val = daip.processInput(xpath, "a  ।  b", null);
         assertEquals("U+0964 DEVANAGARI DANDA with spaces becomes pipe", normVal, val);
     }
+
+    private static final String FSR_START_PATH = "//ldml/personNames/foreignSpaceReplacement";
+    private static final String EMPTY_ELEMENT_VALUE = "❮EMPTY❯";
```

Comment:
is this better than defining it once as public?

Reply:
Will look at that in followup

Reply:
I can combine them.